### ISSUE TITLE
Recommend a compatible FOSS text editor app in Tips&Tricks

### ISSUE THREAD

**Issue by Catfriend1** [Catfriend1/syncthing-android#130]
Quote from @Sjdennis
`If I try opening thisfile.txt from the “recent changes” menu, in QuickEdit I just end up with a new blank “untitled” file. In turboeditor I get an error “No content provider: /filepath/thisfile.txt”. This appears to be a completely separate issue however, probably just a glitch with the “recent changes” menu itself.
I have no problems opening these files in any other way, I just have a problem overwriting them (saving an updated version with the same name). (...) When I select an item in the “recent changes” menu, out of the apps I currently have installed, it gives QuickEdit and Turbo Editor as options to open it. If I choose either option, the file won’t open. QuickEdit fails silently (just opening a blank file), Turbo Editor gives the error I stated above. To be clearer on the general file location, the error is:
No content provider: /storage/emulated/0/path/to/file.txt
If I open the file through a file browser app, or through the “open” dialog on either editor, it opens fine (doesn’t necessarily save properly as per my other problems, but that’s a separate issue).
I also have another text editor installed, Quoda. This also opens the files fine - but is not given as an option to select when I open a file from the recent changes menu, so I don’t know how it would behave here.
All three apps are from the Play Store, and are free (in terms of money anyway), you’ll find them by name.)`

Forum link:
https://forum.syncthing.net/t/android-cannot-edit-synced-text-files-resource-acquired-but-never-released/12496/7

**Comment by Catfriend1** [comment]
@sjdennis I've just noticed there is a new app on F-Droid called "Editor". Tried it on my phone but unfortunately it crashes when trying to open a text file from Syncthing Recent Activities List.

**Comment by Catfriend1** [comment]
It does work with a weird workaround:
- I've installed Root Explorer (without having root, gplay)
- I've installed Simple File Manager (from Tibor Kaputa, f-droid)
I've created the text file "/storage/emulated/0/1.txt" with Root Explorer and then clicked it to "open with" using "Simple File Manager - File Editor" with "always open with this app ticked".
I then went into "Syncthing-Fork/Recent Changes" and clicked on "1.txt" opening with "Root Explorer". I could then open and edit the text file successfully.
Maybe this can be improved in the future. But anyone knowing to do this would be appreciated filing a PR offering a content provider for text files.
